packages/common/react-components/src/components/Searchbar/Searchbar.tsx

```diff
@@ -25,7 +24,20 @@ export const Searchbar: FC<{
   return (
     <div className='flex flex-1 flex-col bg-white'>
       <div className={mx('flex flex-1 items-center p-2', border && 'border rounded')}>
-        <Input className='w-full outline-0' spellCheck={false} onChange={handleChange} placeholder='Search...' />
+        <Input
+          label={undefined}
+          onChange={(event) => handleChange(event.target.value)}
+          placeholder='Search...'
```

Comment:
This should be a call to `t`, e.g. `t('search placeholder')`, and the string should go in `en-US.ts`.

Reply:
If you’re unfamiliar with `t`, look for `useTranslation` and how components are using it, or feel free to DM me.